src/assets/pos.ts

```diff
@@ -158,12 +164,13 @@ export const createGenesisDataObj = async (
 	accounts: Account[],
```

Comment:
Please sort the array lexicographically (by `stakers.address`) before returning `stakers` on L160 within the `createStakersArray` method.